### ISSUE TITLE
[Accessibility] Expense Report: Talkback does not announce the state of Dropdown controls after activation present under 'Expense' pane.

### ISSUE THREAD

**Issue by RebeccaAnne** [microsoft/AdaptiveCards#6260]
### Target Platforms

Android

### SDK Version

Application Version: 2021.06.04.1

### Application Name

Visualizer

### Problem Description

[34056449](https://microsoft.visualstudio.com/OS/_workitems/edit/34056449)

Talkback does not announce the state of Dropdown controls after activation present under 'Expense' pane. Talkback remains silent.

### Card JSON

[ExpenseReport.json](https://github.com/microsoft/AdaptiveCards/blob/main/samples/v1.2/Scenarios/ExpenseReport.json)
